README.md

```diff
@@ -73,6 +74,7 @@ $ xcaddy build [<caddy_version>]
 
 - `--output` changes the output file.
 - `--with` can be used multiple times to add plugins by specifying the Go module name and optionally its version, similar to `go get`. Module name is required, but specific version and/or local replacement are optional.
+- `--replace` can be used multiple times to replace dependencies to specific forks or local replacements. Useful in development environment to fix bugs in dependencies.
```

Comment:
```suggestion
- `--replace` can be used multiple times to replace dependencies to specific forks or local replacements. Useful in development environment when trying to fix bugs in Caddy's dependencies, via a fork or local checkout.
```

Reply:
What about "--replace is like --with, but does not add a blank import to the code; it only writes a replace directive to go.mod, which is useful when developing on Caddy's dependencies that aren't Caddy modules." ?